apollo-router-core/src/query_planner/mod.rs

```diff
@@ -262,8 +262,8 @@ mod fetch {
                 variables.extend(variable_usages.iter().filter_map(|key| {
                     request
                         .variables
-                        .get(key)
-                        .map(|value| (key.clone(), value.clone()))
+                        .get(key.as_str())
+                        .map(|value| (key.as_str().into(), value.clone()))
```

Comment:
is this to_string() ? or maybe Clone ? 🦋

Reply:
Now that the request is parsed with serde_json_bytes too, that could be a clone

Reply:
trying it out in 58a8be7